tensorflow/compiler/jit/mark_for_compilation_pass_test.cc

```diff
@@ -465,11 +465,8 @@ TEST(XlaCompilationTest, CyclesWithAllDifferentScopesGlobalJitOverridden) {
 
   // The computation is: C = A + relu(A)
   // where A sits in ScopeA, relu(A) sits in ScopeB, and C sits in ScopeC.
-  // In this case, the GlobalJitLevel overrides the scopes to cluster while
-  // ignoring scopes.
-  EXPECT_EQ(3, clusters.size());
-  EXPECT_EQ(clusters["A"], clusters["B"]);
-  EXPECT_EQ(clusters["A"], clusters["C"]);
+  // In this case, the GlobalJitLevel respects the scopes to cluster.
```

Comment:
Preserving this behavior is important.  We have internal users that rely on `--tf_xla_auto_jit=2` to create maximally large clusters without paying any attention to jit scopes.  Can we change `ClusterScopingPass` to use a different attribute that is always respected by `MarkForCompilationPass` and leave our handling of jit scopes the same?

Reply:
Got it. No problem. Will add a new attribute. Probably name it `_XlaAutoScope`?